src/test/lib/TestHelpers.h

```diff
@@ -46,6 +46,10 @@ QuicAddrSetToDuoNic(
     }
 }
 
+#ifndef QUIC_OFFICIAL_RELEASE
+#define QUIC_API_ENABLE_PREVIEW_FEATURES
+#endif
```

Comment:
We should be able to remove this now. I added it when I was trying to figure out what was going on.

Reply:
removed